tests/test_api_skeleton.py

```diff
@@ -22,7 +24,9 @@
 
 repo_packages = [('', 'pypi', 'pip', '8.1.2'),
                  ('r', 'cran', 'acs', ''),
-                 ('r', 'cran', 'https://github.com/twitter/AnomalyDetection.git', ''),
+                 (
+                 'r', 'cran', 'https://github.com/twitter/AnomalyDetection.git',
+                 ''),
```

Comment:
Sorry, some modifications regarding the style were because of the linter which I am using. I am going to revert it